include/wx/combo.h

```diff
@@ -475,6 +476,8 @@ class WXDLLIMPEXP_CORE wxComboCtrlBase : public wxControl,
     virtual bool SetForegroundColour(const wxColour& colour) wxOVERRIDE;
```

Comment:
Shouldn't those 2 be removed/updated? `wxCompositeWindow` already propagates colours to all children, or do we want to avoid changing the colour of the button?